spec/models/user_spec.rb

```diff
@@ -934,6 +934,58 @@
     end
   end
 
+  describe "#send_welcome_message" do
+    let(:user) {
```

Comment:
I don't think you need a multiline block at all here, single line should be fine.